docs/ref/databases.txt

```diff
@@ -287,8 +287,8 @@ MySQL has a couple drivers that implement the Python Database API described in
 All these drivers are thread-safe and provide connection pooling.
```

Comment:
There are only two drivers now, so perhaps "Both drivers" instead of "All these".